pkg/controllers/repositoryserver/repository.go

```diff
@@ -52,8 +53,21 @@ func (h *RepoServerHandler) connectToKopiaRepository() error {
 	)
 }
 
-func (h *RepoServerHandler) getRepositoryCacheSettings() (contentCacheMB, metadataCacheMB int, err error) {
+func (h *RepoServerHandler) getRepositoryCommonArguments() (configFilePath, logDirectory string) {
+	configFilePath = command.DefaultConfigFilePath
+	logDirectory = command.DefaultLogDirectory
+	if h.RepositoryServer.Spec.Repository.ConfigFilePath != "" {
+		configFilePath = h.RepositoryServer.Spec.Repository.ConfigFilePath
+	}
+	if h.RepositoryServer.Spec.Repository.LogDirectory != "" {
+		logDirectory = h.RepositoryServer.Spec.Repository.LogDirectory
+	}
+	return
+}
+
+func (h *RepoServerHandler) getRepositoryCacheSettings() (contentCacheMB, metadataCacheMB int, cacheDirectory string, err error) {
```

Comment:
```suggestion
func (h *RepoServerHandler) getRepositoryCacheSettings() (cacheSizeSettings CacheSizeSettings, err error) {
```
We should return the struct if we can to avoid returning multiple fields.

Reply:
@ankitjain235 I will address this as a part of [this](https://github.com/kanisterio/kanister/issues/2231) issue where I will be making cacheSizeSettings fields as `*int` , In current scenarios the fields in the struct are strings. Hence lot of conversion - deconversion  is happening, which I am not liking. 
For now I have removed naked returns but still returning multiple fields . WDYT ?